projects/plugins/jetpack/modules/masterbar/inline-help/inline-help.js

```diff
@@ -0,0 +1,20 @@
+( function () {
+	var windowObjectReference = null;
+	function init() {
+		document.querySelector( '[data-faux-inline-help]' ).addEventListener( 'click', function ( e ) {
```

Comment:
Is there any chance of this returning `null` and it erroring on `.addEventListener`?

Reply:
Fixed